package.json

```diff
@@ -21,17 +21,21 @@
   },
   "homepage": "https://github.com/TrendGame/TrendGame#readme",
   "devDependencies": {
+    "babel-core": "^6.24.1",
     "babel-loader": "^7.0.0",
     "babel-preset-es2015": "^6.24.1",
     "babel-preset-react": "^6.24.1",
-    "webpack": "^2.4.1",
-    "babel-core": "^6.24.1"
+    "nodemon": "^1.11.0",
```

Comment:
Did you mean to add `nodemon` as a dependency *and* change line 10 to `node server.js`